custom/enikshay/management/commands/get_vouchers_for_bets.py

```diff
@@ -67,12 +66,6 @@ def handle(self, domain, repeater_id, filename, **options):
                         voucher_case.get_case_property('voucher_fulfilled_by_id'))
                     payload[u'AgencyId'] = agency_user.raw_username
 
-                    approver_id = voucher_case.get_case_property('voucher_approved_by_id')
-                    approver = CommCareUser.get_by_user_id(approver_id)
-                    payload[u'EnikshayApprover'] = approver.name
-                    payload[u'EnikshayRole'] = approver.user_data.get('usertype')
-                    payload[u'EnikshayApprovalDate'] = voucher_case.get_case_property('date_approved')
-
                     payload['Succeeded'] = record.succeeded
```

Comment:
Looks like they don't want this any more, and instead want `PaymentStatus` (you might have fixed in a later commit)